hips/draw/simple.py

```diff
@@ -1,5 +1,156 @@
+        A list of HipsTile
+
+    Returns
+    -------
+    np.ndarray
+        Returns a numpy array containing all HiPS tiles projected onto it
+    """
+    all_sky = np.zeros(geometry.shape)
+    for tile in tiles:
+        draw_tile = DrawSkyImageOneTile(geometry, tile, all_sky.shape)
+        draw_tile.compute_corners()
+        draw_tile.apply_projection()
+        all_sky += draw_tile.warp_image()
+    return all_sky
+
+
+class DrawSkyImageOneTile:
+    """A private function for drawing a tile over a sky image.
+
+    Parameters
+    ----------
+    geometry : 'WCSGeometry'
+        An object of WCSGeometry
+    tile : HipsTile
+       An object of HipsTile
+    shape ; tuple
+        Shape of the all-sky image
+    """
+
+    def __init__(self, geometry: WCSGeometry, tile: HipsTile, shape: tuple) -> None:
+        self.geometry = geometry
+        self.tile = tile
+        self.shape = shape
+        self.wcs = WCS(tile.header)
+        self.corners = None
+        self.pt = None
+
+    def compute_corners(self) -> None:
+        nside = hp.order2nside(self.tile.meta.order)
+        theta, phi = boundaries(nside, self.tile.meta.ipix)
+        radec = SkyCoord(ra=phi, dec=np.pi / 2 - theta, unit='radian', frame='icrs')
+        self.corners = []
+        for i in range(len(radec.ra.deg)):
+            self.corners.append([radec.ra.deg[i], radec.dec.deg[i]])
+
+    def apply_projection(self) -> None:
+        src = self.geometry.wcs.wcs_world2pix(self.corners, 0)
+        dst = np.array([[511, 0], [511, 511], [0, 511], [0, 0]])
+        self.pt = tf.ProjectiveTransform()
+        self.pt.estimate(src, dst)
+
+    def warp_image(self) -> np.ndarray:
+        return tf.warp(self.tile.data, self.pt, output_shape=self.shape)
+
+
+def _fetch_tiles(healpix_pixel_indices: np.ndarray, order: int, hips_survey: HipsSurveyProperties) -> 'HipsTile':
+    """Fetch HiPS tiles from a remote URL.
+
+    Parameters
+    ----------
+    healpix_pixel_indices : np.ndarray
+        A list of HEALPix pixel indices
+    order : int
+        Order of the HEALPix map
+    hips_survey : HipsSurveyProperties
+        An object of HipsSurveyProperties
+
+    Returns
+    -------
+    'HipsTile'
+        Returns an object of  HipsTile
+    """
+    for healpix_pixel_index in healpix_pixel_indices:
+        tile_meta = HipsTileMeta(order=order, ipix=healpix_pixel_index, file_format='fits')
+        tile = HipsTile.fetch(tile_meta, hips_survey.base_url)
+        yield tile
+
+
+def make_sky_image(geometry: WCSGeometry, hips_survey: HipsSurveyProperties) -> np.ndarray:
+    """Make sky image: fetch tiles and draw.
+
+    Parameters
+    ----------
+    geometry : `~hips.utils.WCSGeometry`
+        Geometry of the output image
+    hips_survey : `~hips.HipsSurveyProperties`
+        HiPS survey properties
+
+    Returns
+    -------
+    data : `~numpy.ndarray`
+        Output image pixels
+
+    Examples
+    --------
+    >>> from astropy.io import fits
```

Comment:
Is this example working now?
If no, can you try to put a working example?
Or is there still some functionality missing?